### ISSUE TITLE
Who are contributors anonymous-1234567 and anonymous-13243557?

### ISSUE THREAD

**Issue by benediktahrens** [UniMath/UniMath#1821]
We have contributions by anonymous-1234567 and anonymous-13243557, see https://github.com/UniMath/UniMath/graphs/contributors. Who are these contributors, and how can we ensure they are credited correctly?

**Comment by nmvdw** [comment]
For a while, I committed with alternative e-mail addresses, namely my uni work addresses, and Github connected commits to those e-mail addresses. At some point, I created an anonymous github account to upload the documentation of a paper (https://github.com/anonymous-13243557/EnrichedCats) using `nweide@cs.ru.nl`, and then a part of my commits were connected to that account. As a test, I also made an account using `n.vanderweide@science.ru.nl`, which is 'anonymous-1234567 '. 

I did change the mailmap in git to connect those commits to me, but for some reason, the contributors graph didn't get affected by it.
